Assets/Mirror/Runtime/UNetwork.cs

```diff
@@ -107,7 +108,7 @@ public static byte[] PackMessage(ushort msgType, byte[] content)
             NetworkWriter writer = new NetworkWriter();
 
             // message type (varint)
-            writer.WritePackedUInt32(msgType);
+            writer.WritePackedUInt32((uint)msgType);
```

Comment:
why not write short instead of packed int?
msgType is probably often > 2287, so writepacked would write 3 bytes a lot. probably better to keep it at 2 bytes all the time?

Reply:
ok,  will change

Reply:
good point